Simple Calculator/Calculator.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace Simple_Calculator
+{
+    public partial class Calculator : Form
+    {
+        //variables which are being used
+        string valOne = "";
+        string valTwo = "";
+        string ArithOperator;
+        
+
+        public Calculator()
+        {
+            InitializeComponent();
+        }
+
+        private void button_Click(object sender, EventArgs e)
+        {
+            Button button = (Button)sender;
+            txtResult.Text = txtResult.Text + button.Text;
+        }
+
+        private void operators_Click(object sender, EventArgs e)
+        {
+            valOne = txtResult.Text;
+            Button button = (Button)sender;
+            ArithOperator = button.Text;
+            txtResult.Clear();
+        }
+
+        private void btnEqual_Click(object sender, EventArgs e)
+        {
+            valTwo = txtResult.Text;
+            CalCulateResult();
+        }
+
+        private void Calculator_FormClosing(object sender, FormClosingEventArgs e)
+        {
+            DialogResult result = MessageBox.Show("Want to close Calculator?", "Calculator", MessageBoxButtons.YesNo, MessageBoxIcon.Warning);
+            if (result == DialogResult.Yes)
+            {
+                //Auto Cancel
+            }
+            if (result == DialogResult.No)
+            {
+                e.Cancel = true;
+            }
+        }
+
+        string  CalCulateResult()
+        {
+            switch (ArithOperator)
+            {
+                case "+":
+                    txtResult.Text = (Int32.Parse(valOne) + Int32.Parse(valTwo)).ToString();
```

Comment:
I told you not to set any variable or UI field in this method. But this is exactly what you are doing when you set `txtResult.Text` to some value.

Calculate the result. Place it inside a variable.
And then return the variable.

Reply:
variable created.